py/server/tests/test_udf_scalar_args.py

```diff
@@ -601,5 +602,17 @@ def misuse_c(c: int) -> int:
             with self.assertRaises(DHError) as cm:
                 t.update("V = misuse_c(C)")
 
+    def test_boxed_type_arg(self):
+        def f(p1: float, p2: np.float64) -> bool:
+            return p1 == 0.05
+
+        dv = 0.05
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
Here I see the warnings tests.  I did not see them elsewhere.

Reply:
For a positive test case, this should suffice.